e2e/cypress/e2e/deleteOrderModal.cy.js

```diff
@@ -6,7 +6,7 @@ describe("deleteOrderModal", () => {
     cy.get("h4").contains("Slet reservering?");
     cy.get("[data-cy=delete-order-confirmation-1]").should(
       "have.text",
-      "Er du sikker på, at du vil slette din reservering:"
+      "Du er ved at slette din reservering:"
```

Comment:
Du kan også bruge en lidt løsere sammenligning, og skrive noget i stil med
```
cy.get("[data-cy=delete-order-confirmation-1]")
   .should("contain", "slet")
   .and("contain", "reservering";
```
For at undgå at skulle rette tests når du ændrer formuleringer

Reply:
indtil man laver om i slet og reservering. jeg beholder det for nu.